sklearn/sklearn-template/template/trainer/task.py

```diff
@@ -70,21 +150,59 @@ def _parse_args(argv):
 
   # TODO(cezequiel): Change to read from BigQuery table instead.
   parser.add_argument(
-      '--bq_table',
-      help='Bigquery table containing input dataset.',
-      required=True,
+    '--bq_table',
+    help='Bigquery table containing input dataset.',
+    required=True,
   )
 
   parser.add_argument(
-      '--job-dir',
-      help='Output directory for exporting model and other metadata.',
-      required=True,
+    '--job-dir',
+    help='Output directory for exporting model and other metadata.',
+    required=True,
   )
 
   parser.add_argument(
-      '--log_level',
-      help='Logging level.',
-      default='INFO',
+    '--log_level',
+    help='Logging level.',
+    choices=[
+      'DEBUG',
+      'ERROR',
+      'FATAL',
+      'INFO',
+      'WARN'
+    ],
+    default='INFO',
+  )
+
+  parser.add_argument(
+    '--n_estimator',
+    help='Number of trees in the forest.',
+    default=10,
+    type=int
```

Comment:
Everywhere: add comma to last item of a list when using multi-line form, e.g. `type=int,`

Reply:
Modified as recommended.